packages/core/lib/commands/debug/index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   run: require("./run"),
   meta: require("./meta"),
-  mergeConfigNetwork: require("./mergeConfigNetwork"),
-  loadConfig: require("./loadConfig")
+  mergeConfigNetwork: require("../../mergeConfigNetwork"),
```

Comment:
We should remove these two exports from the `index.js` since they are no longer in this directory and are now shared. `index.js` is generally meant to expose things in the directory in which it is located - think of an index for a book: it tells you what is inside :)

Reply:
You are right @eggplantzzz . We don't need to export these now since they are outside of the `debug` directory.